### DISCUSSION TITLE
Error in ENSEMBLE_STAT_PROB_PCT_THRESH

### DISCUSSION THREAD

**Discussion by AnastasiaBundel** [dtcenter/METplus#2117] [Configuration] [answered]
Hello!
I am getting an error when running EnsembleStat in METplus with
ENSEMBLE_STAT_PROB_PCT_THRESH =  ==0.1 .
If it is set to ENSEMBLE_STAT_PROB_PCT_THRESH =  ==0.25, it runs correctly.

The error is :

ERROR  :.
ERROR  :   AsciiTable::rc_to_n() -> range check error ....
ERROR  :                            (Nrows, Ncols) = (9, 67)
ERROR  :                            (r, c) = (9, 0)
ERROR  :.

In MET, there is no such error, any prob_pct_thresh works well.

Thank you, 
Anastasia.

**Comment by CPKalb** [comment]
Hi Anastasia:

Would you be able to post your configuration file here so we can take a look at it?  You may need to give it a ".txt" extension to be able to upload.

Thanks,
Christina

**Reply by AnastasiaBundel** [reply, marked as answer]
Hi Christina,

I fixed the problem it seems.
My cdf_bins setting in METplus configuration was left equal to 1. 
When I change it to 10, all works fine with prob_pct_thresh = 0.1. 
As far as I understood from the MET Manual, it relates to setting cdf_bins as floats or integers.

I think you can mark the issue as solved.
Thank you, 
Anastasia.
